geniza/corpus/models.py

```diff
@@ -334,6 +333,15 @@ def shelfmark(self):
             )
         )
 
+    @property
+    def shelfmark_display(self):
```

Comment:
this is definitely out of the scope of the issue you were nominally working on! do we have an issue for this? is it worth tracking?

Reply:
it's based on a convo we had at a meeting where we discussed notating joins with the plus sign; i thought it might be helpful to see it on the document detail pages to continue the conversation. you can also think of it as a stubbed implementation of #154; eventually this same function will need to collapse shelfmarks into ranges too.